platform-db-runtime/src/main/java/com/softicar/platform/db/runtime/table/listener/DbTableRowNotificationSet.java

```diff
@@ -22,7 +22,7 @@ public DbTableRowNotificationSet(IDbTable<R, ?> table) {
 	}
 
 	@Override
-	public Collection<DbTableRowNotification<R>> getAllNotifications() {
+	public List<DbTableRowNotification<R>> getAllNotifications() {
```

Comment:
why change this?

Reply:
To imply that elements are contained in a specific order.

Reply:
(this change was reverted)